chaincode/contracts/AssetManager.sol

```diff
@@ -0,0 +1,190 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.9;
+
+// import "hardhat/console.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "./Enums.sol";
+
+contract AssetManager is Ownable {
+    event DepositSuccesful(address user, uint256 amount, uint256 balance);
```

Comment:
Should index user, so we can look up the events later

Reply:
Same for other success events

Reply:
done